resources/public/api/conf/2.0/application.yaml

```diff
@@ -0,0 +1,10973 @@
+          "propertyPurchaseValue": 250000,
+          "supersede": {
+            "originalLifeEventId": "5678900001",
+            "originalEventDate": "2017-10-10"
+          }
+        }       
+        </pre>
+              </td>
+              <td>
+                <p>HTTP status: <code class="code--slim">409 (Conflict)</code></p>
+                <pre class="code--block">
+        {
+          "code": "SUPERSEDED_LIFE_EVENT_ALREADY_SUPERSEDED",
+          "message": "This life event has already been superseded",
+          "lifeEventId": "5678900002"
+        }                        
+        </pre>
+              </td>
+            </tr>
+            <tr>
+              <td>
+                <p>Purchase outcome already exists</p>
+                <p class="code--block"> <strong>lisaManagerReferenceNumber:</strong><br /> <a href="#section/Testing">Use your test user profile</a><br /> <br /> <strong>accountId:</strong><br />0000000409 </p>
+              </td>
+              <td>
+                <pre class="code--block">
+        {
+          "fundReleaseId": "3456789001",
+          "eventDate": "2017-10-10",
+          "propertyPurchaseResult": "Purchase completed",
+          "propertyPurchaseValue": 250000
+        }                                                     
+        </pre>
+              </td>
+              <td>
+                <p>HTTP status: <code class="code--slim">409 (Conflict)</code></p>
+                <pre class="code--block">
+        {
+          "code": "LIFE_EVENT_ALREADY_EXISTS",
+          "message": "The investor’s life event has already been reported",
+          "lifeEventId": "5678900001"
+        }                                               
+        </pre>
+              </td>
+            </tr>
+            <tr>
+              <td>
+                <p>The associated fund release has been superseded</p>
+                <p class="code--block"> <strong>lisaManagerReferenceNumber:</strong><br /> <a href="#section/Testing">Use your test user profile</a><br /> <br /> <strong>accountId:</strong><br />2000000409 </p>
+              </td>
+              <td>
+                <pre class="code--block">
+        {
+          "fundReleaseId": "3456789000",
+          "eventDate": "2017-10-05",
+          "propertyPurchaseResult": "Purchase completed",
+          "propertyPurchaseValue": 250000
+        }
+        </pre>
+              </td>
+              <td>
+                <p>HTTP status: <code class="code--slim">409 (Conflict)</code></p>
+                <pre class="code--block">
+        {
+          "code": "FUND_RELEASE_SUPERSEDED",
+          "message": "This fund release has already been superseded",
+          "lifeEventId": "3456789001"
+        }                                               
+        </pre>
+              </td>
+            </tr>
+          </thead>
+        </table>
+      operationId: reportOutcomeOfPropertyPurchase
+      parameters:
```

Comment:
This is also missing a Content-Type header (the RAML is incorrect) as shown in the snippet below.

<img width="637" alt="Screenshot 2023-02-09 at 11 14 10" src="https://user-images.githubusercontent.com/63714619/217836735-df4d47f7-02f5-4694-b9c7-556900faa033.png">

Please correct this at least in this PR by adding the reference `- $ref: '#/components/parameters/contentTypeHeader'` as making this endpoint call without the header results in a 400. Thanks. This is shown in the snippets below

<img width="963" alt="Screenshot 2023-02-10 at 08 46 55" src="https://user-images.githubusercontent.com/63714619/218045654-318abdaf-b790-430e-8f0b-5144ccd714dc.png">

<img width="966" alt="Screenshot 2023-02-10 at 08 45 36" src="https://user-images.githubusercontent.com/63714619/218045799-1553fbc1-449e-4edd-8ae4-9e9dac584308.png">